tic_tac_toe/game.py

```diff
@@ -63,22 +80,55 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
+    win = False
+    for i in range(3): #Rows -- i know this is not pythonic -- thinking about how to improve    
```

Comment:
You are on the right track. If you're looking for a more pythonic solution, then list comprehension might be a good option to try.